tutorials/36_Building_Fallbacks_with_Conditional_Routing.ipynb

```diff
@@ -0,0 +1,563 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    Answer the following query given the documents retrieved from the web.
</pre><p>Follow up by saying. 'Your answer shoud indicate that your answer was generated from websearch'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/287/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>